src/IIIFPresentation/API/Features/Manifest/DlcsManifestCoordinator.cs

```diff
@@ -188,28 +229,76 @@ await dlcsApiClient.UpdateAssetManifest(dbManifest.CustomerId,
         
         assetsTrackedElsewhere.AddRange(dlcsAssets.Select(a => a.GetAssetId(customerId)));
 
-        if (assetsTrackedElsewhere.Count == assets.Count)
+        if (assetsTrackedElsewhere.Count == assets.Count && !trackedAssetsToReingest.Any())
         {
             logger.LogTrace("All assets tracked for {ManifestId}", dbManifest?.Id ?? "new manifest");
-            return ([], assetsTrackedElsewhere);
+            return new AssetsWithAdditionalWork([], assetsTrackedElsewhere, []);
         }
-        
-        var untrackedAssets = GetUntrackedAssets(assets, assetsInDatabase, assetsTrackedElsewhere);
-        
-        return (untrackedAssets, assetsTrackedElsewhere);
+
+        return null;
     }
 
+    private (List<CanvasPainting> AssetsInDatabase, List<CanvasPainting> TrackedAssetsToReingest) RetrieveTrackedAssets(
+        int customerId, List<AssetIdsWithReingestion> assetIdsWithReingestion)
+    {
+        List<CanvasPainting> assetsInDatabase = [];
+        List<CanvasPainting> trackedAssetsToReingest = [];
 
-    private static List<JObject> GetUntrackedAssets(List<JObject> payloadAssets, List<CanvasPainting> assetsInDatabase, 
-        List<AssetId> dlcsAssetIds)
+        foreach (var canvasPainting in dbContext.CanvasPaintings.Where(cp => cp.CustomerId == customerId))
```

Comment:
This is really expensive. Looking at UoL, we have 150K canvasPaintings, could easily end up over 1M so we don't want to iterate over every one every time.

Be more targeted? 

First start `dbContext.CanvasPaintings.Where(cp => cp.CustomerId == customerId && cp.Manifest = manifestId && cp.AssetId != null)` (do we have these already)? I suspect that would catch the 99% of cases. 

If we don't find them then make equivalent of `select * from canvas_painting where customer = @id and assetId in [@id_list]`. Will leverage database indexes etc. Depending on the number of assets we may need to batch this